sdk/mcp/README.md

```
@@ -0,0 +1,122 @@
+
+- **30 resources** — API reference (TS + Go), protocol docs, security patterns, examples (TS + Go), use cases, migration
+- **8 tools** — method lookup, type lookup, search, RPC format, import validation, concept explanation, scaffolding (TS + Go)
+- **3 prompts** — guided workflows for building apps (covers both TS and Go), migrating from v0.5.3, building AI agents
+
+---
+
+## Tools
+
+### `lookup_method`
+
+Look up a Client method by name. Supports an optional `language` parameter.
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)